survivalstan/utils.py

```diff
@@ -258,10 +258,13 @@ def prep_pp_survival_data(models, time_element='y_hat_time', event_element='y_ha
                          time_col='event_time', event_col='event_status',
                          by=None,
                          **kwargs):
-    pp_data = prep_pp_data(models, time_element=time_element, event_element=event_element, time_col=time_col, event_col=event_col, **kwargs)
+    pp_data = prep_pp_data(models, time_element=time_element,
+                           event_element=event_element, time_col=time_col, event_col=event_col, **kwargs)
     groups = ['iter', 'model_cohort']
-    if by:
+    if by and isinstance(by, str):
```

Comment:
Maybe add a docstring explaining that `by` can be either a `str` or `list`? Otherwise LGTM!